src/events/requestVariableEvent.ts

```diff
@@ -0,0 +1,13 @@
+import { EventEmitter } from "vscode";
+import { RequestVariableCacheKey } from "../models/requestVariableCacheKey";
+
+interface RequestVariableEvent {
+  cacheKey: RequestVariableCacheKey;
+}
+
+const eventEmitter = new EventEmitter<RequestVariableEvent>();
+
+export const fireEvent = (event: RequestVariableEvent) =>
+  eventEmitter.fire(event);
```

Comment:
Do we really need the `RequestVariableEvent`, if so, I think we should also export the `RequestVariableEvent`, otherwise, no one can use it in their own listeners.

Reply:
I'll export it, we need some type to use with `EventEmitter`. One idea which I didn't get time to complete was checking a specific document rather than "all" when firing the event. In that case the cacheKey is useful to send with the event.

Reply:
@cbrevik that's OK, we can improve it later. Currently it's OK for me.